### ISSUE TITLE
Is this Project dead?

### ISSUE THREAD

**Issue by axaluss** [scalatron/scalatron#66]
Is this Project dead?

**Comment by tlasica** [comment]
I still find it quite a good fun so I decided to prepare a java 8 / scala 2.10 version.

**Comment by AmirKarimi** [comment]
Nice work @tlasica, it deserves more attention.

**Comment by jtjeferreira** [comment]
thx @tlasica

**Comment by murmelsson** [comment]
+1 for @tlasica - spent an hour working out how to get the scalatron source code to build using the old versions of eveything (inc Java 1.7), then stumbled on Tomek's java1.8/scala2.10 version.

**Comment by barkhorn** [comment]
spending some time to move this to a newer version of Akka and Scala 2.11 at the moment on my fork if anyone is curious. https://github.com/barkhorn/scalatron

**Comment by scalatron** [comment]
Well, the project died because I got sidetracked for a few weeks and then through an unfortunate sequence of events lost my passwords to the github account and all linked accounts. I had intended this project to be a kind of anonymous donation to the Scala community, and there wasn't really an easy way to recover the various account without giving that up. Now, through a similarly miraculous sequence of events, however, I found my passwords... almost six years later! Obviously I'm super-far away from the code now, but if anybody has any sensible ideas on what to do with the project or would like to take it over or bring it up-to-date, I'm open to suggestions.
